### PR TITLE
Add focusable button and optional play label

### PR DESCRIPTION
**Pull request by spacedawwwg**
addresses #9

### CODE REVIEW

**Comment by spacedawwwg** [comment]
@paulirish separated the accessible play button code here and closed the previous PR

File: src/lite-yt-embed.css [hunk 1]
```diff
@@ -49,13 +49,25 @@ lite-youtube > .lty-playbtn {
...
+    border: none;
+    margin: 0;
+    padding: 0;
+    overflow: visible;
+    color: inherit;
+    font: inherit;
+    line-height: normal;
+    -webkit-appearance: none;
+    cursor: pointer;
+    outline: none;
```
Comment (paulirish) [inline src/lite-yt-embed.css:66]:
```suggestion
```

I don't believe we need any of these. It'll still show up visually how we want. (tested in chrome/ff/safari on mac. didnt test windows/android/ios)

also.. looking at the real embed... https://paulirish.github.io/lite-youtube-embed/variants/yt.html the button is focusable and gets a real outline... so i do think we want to keep the default `outline` on this.. yeah?

File: src/lite-yt-embed.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ class LiteYTEmbed extends HTMLElement {
         // Gotta encode the untrusted value
         // https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#rule-2---attribute-escape-before-inserting-untrusted-data-into-html-common-attributes
         this.videoId = encodeURIComponent(this.getAttribute('videoid'));
+        this.playLabel = this.getAttribute('playlabel') ? encodeURIComponent(this.getAttribute('playlabel')) : 'Play';
```
Comment (paulirish) [inline src/lite-yt-embed.js:18]:
```suggestion
        const playLabel = this.getAttribute('playlabel');
        this.playLabel = playLabel ? encodeURIComponent(playLabel) : 'Play';
```

nothing big changed here. just style and touching the DOM less.

i really like the default label of `Play` and that it matches YT's real one: 

image

File: src/lite-yt-embed.css [hunk 3]
```diff
@@ -49,13 +49,25 @@ lite-youtube > .lty-playbtn {
...
 }
 
-lite-youtube:hover > .lty-playbtn {
+lite-youtube:hover > .lty-playbtn,
+lite-youtube .lty-playbtn:focus {
```
Comment (paulirish) [inline src/lite-yt-embed.css:70]: yahh! red on focus! 

interestingly, the real one doesnt do this, but i agree with your executive decision.  :D

**Review by paulirish — APPROVED** [review]
Great stuff.  I have a few tweaks I think I'll just land so we can land + ship this. 

(if i wait a day i'm scared I'll take weeks/months again! 😢   )

**Comment by paulirish** [comment]
@spacedawwwg this PR is SO good. the move to a `<button>` is 100% the right one and i'm really glad you helped us get there. cheers.

**Pull request merged**